tokio-util/src/task/join_map.rs

```diff
@@ -96,11 +92,9 @@ use tokio::task::{AbortHandle, Id, JoinError, JoinSet, LocalSet};
 /// ```
 ///
 /// [`JoinSet`]: tokio::task::JoinSet
-/// [unstable]: tokio#unstable-features
 /// [abort]: fn@Self::abort
 /// [abort_matching]: fn@Self::abort_matching
 /// [contains]: fn@Self::contains_key
-#[cfg_attr(docsrs, doc(cfg(all(feature = "rt", tokio_unstable))))]
 pub struct JoinMap<K, V, S = RandomState> {
```

Comment:
Can we add back the doc attribute? Right now it is rendered like this:

> Available on crate feature join-map and (crate features rt or join-map) only.

I believe we can improve this with an explicit attribute.
```
#[cfg_attr(docsrs, doc(cfg(feature = "join-map")))]
```